oak_runtime/src/lib.rs

```diff
@@ -208,6 +217,19 @@ struct NodeInfo {
     node_stopper: Option<NodeStopper>,
 }
 
+impl NodeInfo {
+    /// Returns a unique debug_id used to identify the node in the debug output,
+    /// consisting out of the provided [`NodeId`], and the node's name.
```

Comment:
```suggestion
    /// consisting of the provided [`NodeId`], and the node's name.
```
Or is `consist out of` standard usage?